clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientCloudBuilderStep1.java

```diff
@@ -15,6 +15,7 @@
  */
 package io.camunda.zeebe.client;
 
+@Deprecated
```

Comment:
❓ Should we add some Javadoc to hint at the replacing class and the involved versions here, e.g. deprecated with 8.6 for removal with 8.8? We should then add this for all such depreciations.

Reply:
Yes, I think it make sense to add it for every deprecation.